tests/app/main/views/test_search.py

```diff
@@ -103,3 +56,41 @@ def test_forms_visible_for_ccs_data_controller_role(self):
         assert "Find a supplier by name" in question_headings
         assert "Find a supplier by DUNS number" in question_headings
         assert "Find a supplier by company registration number" in question_headings
+
+    @pytest.mark.parametrize(
+        "role, supplier_name_form_should_exist, supplier_duns_form_should_exist, service_id_form_should_exist, "
+        "company_reg_number_should_exist", [
+            ("admin", True, True, True, True),
+            ("admin-ccs-category", True, True, True, True),
+            ("admin-ccs-sourcing", True, True, False, True),
+            ("admin-framework-manager", True, True, True, True),
+            ("admin-ccs-data-controller", True, True, False, True)
+        ])
+    def test_forms_visible_for_relevant_roles(
```

Comment:
This test is nicely comprehensive 👍 so I think we can delete the one above it, `test_forms_visible_for_ccs_data_controller_role` 🔥 😸